### PR TITLE
Load ZH

### PR DESCRIPTION
**Pull request by profxj**
Enables ZH loading from JSON dicts
in AbsSystem

Plus a little refactoring..

### CODE REVIEW

File: linetools/isgm/abssystem.py [hunk 1]
```diff
@@ -226,6 +208,10 @@ def __init__(self, radec, zabs, vlim, zem=0., abs_type=None,
         # Kinematics
         self.kin = {}
 
+        # Metallicity
```
Comment (jessicawerk) [inline linetools/isgm/abssystem.py]: The attribute in the full code is called MH, and this is ZH. I can't recall if M/H and Z/H are different quantities (this always confuses me), but I think we should be consistent in the naming.

**Comment by jessicawerk** [comment]
Other than the quibble between MH and ZH, This looks good to me.

File: linetools/isgm/abssystem.py (hunk 1 above)
Comment (profxj) [inline linetools/isgm/abssystem.py]: Good catch.  

We go with ZH

**Comment by profxj** [comment]
Merging

**Pull request merged**